src/windows/AppVersionProxy.js

```diff
@@ -0,0 +1,19 @@
+AppVersionProxy = {
+  getVersionNumber: function (successCallback, failCallback, args) {
```

Comment:
This method not work on Windows Phone 8.1 (NOKIA Lumia 625).
I've changed that works:
```
  getAppName: function (successCallback, failCallback, args) {
      Windows.ApplicationModel.Package.current.installedLocation.getFileAsync("AppxManifest.xml").then(function (file) {
          Windows.Data.Xml.Dom.XmlDocument.loadFromFileAsync(file).then(function (xdoc) {
              var displayName = xdoc.getElementsByTagName("DisplayName");
              if (displayName && displayName.length === 1) {
                  var name = displayName[0].innerText;
                  successCallback(name);
              } else {
                  failCallback({ code: -1, message: "ERR_DISPLAY_NAME_NOT_FOUND" });
              }
          }, failCallback);
      }, failCallback);
  }
```